lib/p5.play.js

```diff
@@ -1213,23 +1237,8 @@ function Sprite(pInst, _x, _y, _w, _h) {
    * Keep animation properties in sync with how the animation changes.
    */
   this._syncAnimationSizes = function(animations, currentAnimation) {
```

Comment:
`Sprite._syncAnimationSizes` (called only from within `Sprite.update`) used to do double-duty, adjusting the sprite dimensions and (sometimes) the collider dimensions... and then `Sprite.update` would do more adjusting of the collider dimensions too.  This method has been reduced to "update the sprite dimensions" and colliders and handled later.